src/packs/hipaa-security.ts

```diff
@@ -750,7 +750,7 @@ export class HIPAASecurityChecks extends NagPack {
       node: node,
     });
     this.applyRule({
-      info: 'The RDS DB instance does not enhanced monitoring enabled - (Control ID: 164.312(b)).',
```

Comment:
Same request for this one

1. https://github.com/quixoticmonk/cdk-nag/blob/rules_info_wording/src/packs/nist-800-53-r4.ts#L626
2. https://github.com/quixoticmonk/cdk-nag/blob/rules_info_wording/src/packs/nist-800-53-r5.ts#L696

Reply:
All of these are updated as requested.